src/Tizen.NUI/src/public/BaseComponents/TextConstants.cs

```diff
@@ -546,6 +547,74 @@ public struct TextFit : IEquatable<TextFit>
         public override int GetHashCode() => (Enable, MinSize, MaxSize, StepSize, FontSizeType, FontSize).GetHashCode();
     }
 
+    /// <summary>
+    /// A struct to pass data of TextFitArray's OptionList.
+    /// </summary>
+    /// <remarks>
+    /// The TextFitArrayOption struct is used as an item to TextFitArray's OptionList. <br />
+    /// See <see cref="Tizen.NUI.Text.TextFitArray"/>.
+    /// </remarks>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1815: Override equals and operator equals on value types")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct TextFitArrayOption
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TextFitArrayOption(float pointSize)
+        {
+            PointSize = pointSize;
+            MinLineSize = null;
+        }
+
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TextFitArrayOption(float pointSize, float? minLineSize)
```

Comment:
Just my curiosity :
Is there any reason to check null only for the type of `minLineSize`?
`pointSize` is not checked for Null?

Reply:
In TextFitOption,
`PointSize` is essential, but `MinLineSize` is optional,so I used nullable.
Is there a better way?

Reply:
Understood.
As you mentioned, the constructor with the essential parameter exists above.
And no, I think you can go the way you created it just like now :)
Thank you for answering it.